### DISCUSSION TITLE
Login Page

### DISCUSSION THREAD

**Discussion by jaredrampartap** [openbroadcaster/observer#35] [Q&A]
Hi, I've gone through the installation process successfully and I'm able to get to the login page but after putting in my credentials the page just refreshes instead of taking me further. Is there anything I can check in order to fix this? In the apcahe config file I have the root directory as DocumentRoot /var/www/html/observer/ and changed the ownership to www-data.

**Comment by radiorob** [comment]
Use new public installer https://github.com/openbroadcaster/observer/blob/main/ob.installer.sh

**Reply by mamimsllc** [reply]
After using the new public installer I am still getting the same problem also

**Reply by brookgagnon** [reply]
> After using the new public installer I am still getting the same problem also

Have you run all updates in "/updates"? If so, can you check the server logs for PHP errors?

**Reply by mamimsllc** [reply]
That worked.  Thank you so much

**Reply by brookgagnon** [reply]
Great! :) Next version will actually show an error when updates are required. This one tripped up a lot of users.
